### PR TITLE
User can select begin and end date through a calendar

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
_As a user,
in order to select dates in a more user-friendly way
I would like to be able to set recording dates through a calendar_

Changes proposed in this pull request:

* add react-datepicker package
* implement selection of dates through calendar in begin and end input fields

#### What I have learned working on this feature: 
How to implement a more user-friendly date selection through calendar in input fields

#### Screenshots:
image

### CODE REVIEW

File: src/components/timeTrackingTable.js [hunk 1]
```diff
@@ -314,74 +324,78 @@ export class TimeTrackingTable extends Component {
...
+										selection
+										defaultValue=""
+										options={taskOptions}
+										onChange={(e, { value }) => this.handleActivityChange(value)}
+									/>
+								</Table.Cell>
+								<Table.Cell>{saveButton}</Table.Cell>
+							</Table.Row>
+							{listEntries}
+						</Table.Body>
```
Comment (vick3d) [inline src/components/timeTrackingTable.js]: remove space

File: src/components/timeTrackingTable.js [hunk 2]
```diff
@@ -314,74 +324,78 @@ export class TimeTrackingTable extends Component {
 							</Table.Row>
 						</Table.Header>
```
Comment (vick3d) [inline src/components/timeTrackingTable.js]: remove space

File: src/components/timeTrackingTable.js (hunk 1 above)
Comment (amandagibson) [reply src/components/timeTrackingTable.js]: done

File: src/components/timeTrackingTable.js (hunk 2 above)
Comment (amandagibson) [reply src/components/timeTrackingTable.js]: done

**Pull request merged**